src/libs/PolicyUtils.ts

```diff
@@ -8,6 +8,22 @@ type MemberEmailsToAccountIDs = Record<string, number>;
 type PersonalDetailsList = Record<string, PersonalDetails>;
 type UnitRate = {rate: number};
 
+let allPolicyTags: OnyxCollection<PolicyTags> = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY_TAGS,
+    waitForCollectionCallback: true,
+    callback: (value) => {
+        if (!value) {
+            return;
+        }
+        allPolicyTags = Object.fromEntries(Object.entries(value).filter(([, policyTags]) => !!policyTags));
```

Comment:
Is `[, policyTags]` right? I've never seen that before so it looks like a mistake. Maybe it's something with TypeScript I am not familiar with.

Reply:
ugh! fixed.